agent/cmd/pmm-agent-entrypoint/main.go

```diff
@@ -71,15 +71,15 @@ var (
 var pmmAgentProcessID = 0
 
 func runPmmAgent(ctx context.Context, commandLineArgs []string, restartPolicy restartPolicy, l *logrus.Entry, pmmAgentSidecarSleep int) int {
-	pmmAgentFullCommand := "pmm-admin " + strings.Join(commandLineArgs, " ")
+	pmmAgentFullCommand := "pmm-agent " + strings.Join(commandLineArgs, " ")
```

Comment:
Is this safe? What if I will do something like:
pmm-agent | mkdir dddd

Reply:
These arguments can be passed only when you run the container, so if you control the container, then you can run all these commands manually.

Reply:
Ok thanks.